tests/pipeline/test.nf.test.snap

```diff
@@ -0,0 +1,76 @@
+{
+    "input": {
+        "content": [
+            "Samplesheet.tsv:md5,dbf8d1a2b7933dab9e5a139f33c2b1f4",
+            "Metadata.tsv:md5,060b56528bb566eed71f6dfdb52cc395"
+        ],
+        "timestamp": "2023-05-28T20:55:32+0000"
+    },
+    "SBDI": {
+        "content": [
+            "dna.tsv:md5,46bb52a05204ade2d571ac089f732012",
+            "emof.tsv:md5,52ff7d226c79b1f78d5f4895d748f915",
+            "event.tsv:md5,d275c5c0d7dbf7a5b2c61082fbce486d"
+        ],
+        "timestamp": "2023-05-28T20:55:32+0000"
+    },
+    "cutadapt": {
+        "content": [
+            "cutadapt_summary.tsv:md5,5d02749984a811479e7d534fda75163f"
+        ],
+        "timestamp": "2023-05-28T20:55:32+0000"
+    },
+    "software_versions": {
+        "content": [
+            "{BARRNAP={barrnap=0.9}, CUSTOM_DUMPSOFTWAREVERSIONS={python=3.11.0, yaml=6.0}, CUTADAPT_BASIC={cutadapt=3.4}, DADA2_DENOISING={R=4.1.1, dada2=1.22.0}, DADA2_FILTNTRIM={R=4.1.1, dada2=1.22.0}, DADA2_QUALITY1={R=4.1.1, ShortRead=1.52.0, dada2=1.22.0}, DADA2_TAXONOMY={R=4.1.1, dada2=1.22.0}, FASTQC={fastqc=0.11.9}, FILTER_LEN_ASV={Biostrings=2.58.0, R=4.0.3}, FILTER_STATS={pandas=1.1.5, python=3.9.1}, QIIME2_INSEQ={qiime2=2022.11.1}, RENAME_RAW_DATA_FILES={sed=4.7}, SBDIEXPORT={R=3.6.3}, TRUNCLEN={pandas=1.1.5, python=3.9.1}, Workflow={nf-core/ampliseq=2.6.0dev}}"
```

Comment:
Wouldnt Workflow={nf-core/ampliseq=2.6.0dev} fail whith a new version, should that be in here?

Reply:
Changes to release check list after CI nf-test migration:
- Along with bumping the version number of the dev branch to a release version, also bump the version number of the nf-core/pipeline in snapshots (`Workflow={nf-core/fetchngs=1.11.0dev}}`). This is to ensure that the nf-core/pipeline version in the snapshots is the same as the version in the release branch. Currently to be done by manually deleting the `dev` from the version number in the snapshots. But, in future we can have the tools bump version command to do this for us.

Reply:
> Can "content": only have a part of the content or does it need to be complete?

I didn't understand this. could you rephrase?

Reply:
> Currently to be done by manually deleting the dev from the version number in the snapshots. But, in future we can have the tools bump version command to do this for us.

Great, thanks!

> Can "content": only have a part of the content or does it need to be complete?

rephrase: 
Could we use something like
```
"content": [
            "{BARRNAP={barrnap=0.9}, CUSTOM_DUMPSOFTWAREVERSIONS={python=3.11.0, yaml=6.0}, CUTADAPT_BASIC={cutadapt=3.4}, DADA2_DENOISING={R=4.1.1, dada2=1.22.0}, DADA2_FILTNTRIM={R=4.1.1, dada2=1.22.0}, DADA2_QUALITY1={R=4.1.1, ShortRead=1.52.0, dada2=1.22.0}, DADA2_TAXONOMY={R=4.1.1, dada2=1.22.0}, FASTQC={fastqc=0.11.9}, FILTER_LEN_ASV={Biostrings=2.58.0, R=4.0.3}, FILTER_STATS={pandas=1.1.5, python=3.9.1}, QIIME2_INSEQ={qiime2=2022.11.1}, RENAME_RAW_DATA_FILES={sed=4.7}, SBDIEXPORT={R=3.6.3}, TRUNCLEN={pandas=1.1.5, python=3.9.1}}"
```
e.g. without the `Workflow={nf-core/ampliseq=2.6.0dev}`? Therefore the workflow version wouldn't be needed here (Anyway not needed imho. I'd actually remove the complete test of software_versions, I dont really see why I would need to test that. All that software is containerized, if I intentionally change a container I also intentionally change the software versions).

Reply:
We actually remove the nextflow version from the software versions `assert snapshot(UTILS.removeNextflowVersion("$outputDir")).match("software_versions")` so the snapshots can match with different nextflow versions used (22.10.1/latest). That said, I believe recording the software versions including the workflow version in a snapshot is great way to ensure we have a record of the version used for testing and creating the snapshot.